modules/sync/syncs/httpsynchronization.cpp

```diff
@@ -29,13 +29,17 @@
 #include <openspace/util/httprequest.h>
 #include <ghoul/ext/assimp/contrib/zip/src/zip.h>
 #include <ghoul/logging/logmanager.h>
+#include <fstream>
 #include <unordered_map>
 
 namespace {
     constexpr std::string_view _loggerCat = "HttpSynchronization";
 
     constexpr int ApplicationVersion = 1;
 
+    constexpr std::string_view _ossyncVersionNumber = "1.0";
+    constexpr std::string_view _synchronizationToken = "Synchronized";
```

Comment:
```suggestion
    constexpr std::string_view OssyncVersionNumber = "1.0";
    constexpr std::string_view SynchronizationToken = "Synchronized";
```

Since we do `constexpr` variables with CamelCase